gravitee-apim-console-webui/src/components/alerts/alert/triggers/trigger-api-hc-endpoint-status-changed.html

```diff
@@ -16,3 +16,14 @@
 
 -->
 <span style="font-style: italic">This rule does not require any configuration.</span>
+<span style="font-style: italic"
+  >However, it is possible to define an aggregation to receive an alert per api endpoint instead of an unique alert in case of multiple
```

Comment:
```suggestion
  >However, it is possible to define an aggregation to receive an alert per api endpoint instead of an unique alert in case multiple
```
I may be wrong, but seems better like that to me